packages/playground/src/routes/workflows.$workflowId.tsx

```diff
@@ -120,7 +124,7 @@ function WorkflowDetail() {
 
   if (loadingError || parsingError) {
     return (
-      <div className="grid place-items-center h-full">
+      <div className="grid h-full place-items-center">
```

Comment:
Why is the order changed? Is that because of an extension? If so, what are you using?

Reply:
yep it's [Headwind](https://github.com/heybourn/headwind)